atomic_reactor/utils/koji.py

```diff
@@ -258,6 +296,16 @@ def get_koji_module_build(session, module_spec):
     return build, rpm_list
 
 
+def get_maven_components(workflow):
+    """
+    Get list of maven components generated by post_generate_maven_metadata
+    :param workflow: DockerBuildWorkflow instance
+    :return: list, maven components
```

Comment:
@hjmodi; We're not super-consistent about this ourselves, but we seem to moving towards e.g.

```suggestion
    # e.g. a short text explanation of what we return
    :return: A list of maven components. Each component is blah blah
    # e.g. a Python type(s)
    :rtype: list[]  # What is a maven component's type?
```

Reply:
Added info.